lib/client-assets.php

```diff
@@ -655,7 +655,7 @@ function gutenberg_extend_settings_block_patterns( $settings ) {
  * @return array Filtered editor settings.
  */
 function gutenberg_extend_settings_custom_line_height( $settings ) {
-	$settings['__experimentalEnableCustomLineHeight'] = get_theme_support( 'experimental-line-height' );
+	$settings['enableCustomLineHeight'] = get_theme_support( 'custom-line-height' );
```

Comment:
I was hesitant whether to call this "line-height" or "custom-line-height" I went with custom for consistency with other theme support flags (units, spacing).